news/_posts/core-monthly/2022-12-05-core-monthly-2022-11-01-2022-11-30.md

```diff
@@ -0,0 +1,436 @@
+
+Looking forward to meeting you at our next meetings.
+
+#### Public Demo
+
+The eighth demo of the year was held - this time with two presentations from project members. If you haven't watched it already, make sure to [read the summary](/news/public-demo-8-2022/).
+
+#### Committers
+
+In November, [three new committers were welcomed](https://github.com/PrestaShop/open-source/pull/141). An interesting development is a decision that was taken for committers [to act as code owners](https://github.com/PrestaShop/PrestaShop/pull/30377). What does it mean? Until then, the rules were as follows: a Pull Request to the Core had to be approved by at least one maintainer. Now, two committers can fully approve it. This seemingly small step is significant for the entire project.
+
+If you are an active member of the project and want to join as a committer, you can read how to do this [on the article](https://www.prestashop-project.org/maintainers-guide/how-to-become-a-committer/) on the project's website.
+
+#### Focus on code changes
+
+With PrestaShop 8.0 released a month ago, a feature freeze is already getting closer for PrestaShop 8.1. There are a few major subjects for this version that project members are working on.
+
+**Product page**
+
+The first subject is a new product page. The team responsible for this project is working hard to make all the needed changes in the code.
+
+Ongoing work on [the unification of some commands](https://github.com/PrestaShop/PrestaShop/pull/30381) and some refactoring, crucial for the page to be more efficient: so developers can use reliably created commands. In almost every Pull Request related to the new product page, you may notice several new or modified [automated tests](https://github.com/PrestaShop/PrestaShop/pull/30248/files#diff-8f4440222a254dfc66f71464d4d693250a4e2367a1d7a808df6fd308948af694). This should help the team deliver a  product page more stable than ever.
+
+You can also find some Pull Requests about improving or [adding multi store compatibility to the new product page](https://github.com/PrestaShop/PrestaShop/pull/28395).
+
+**New features coming to PrestaShop 8.1**
+
+In November, the community created a few outstanding Pull Requests (which are now merged) with features that are going to be available in the next minor version. Starting with **three (!)** changes from @Hlavtox. He added a new setting to decide if [order should switch to backorder state](https://github.com/PrestaShop/PrestaShop/pull/29753). The second is the highly requested possibility of adding availability labels to each combination. In PrestaShop 8.1, this will be possible [thanks to Daniel and his changes](https://github.com/PrestaShop/PrestaShop/pull/29411). Last but not least, [more options for discontinued products](https://github.com/PrestaShop/PrestaShop/pull/29720) should help merchants better manage such articles in PrestaShop.
```

Comment:
```suggestion
In November, the community created a few outstanding Pull Requests (which are now merged) with features that are going to be available in the next minor version. Starting with **three (!)** changes from @Hlavtox. He added a new setting to decide if [the order should switch to backorder state](https://github.com/PrestaShop/PrestaShop/pull/29753). The second is the highly requested possibility of adding availability labels to each combination. In PrestaShop 8.1, this will be possible [thanks to Daniel and his changes](https://github.com/PrestaShop/PrestaShop/pull/29411). Last but not least, [more options for discontinued products](https://github.com/PrestaShop/PrestaShop/pull/29720) should help merchants better manage such articles in PrestaShop.
```